aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -109,8 +109,26 @@ std::tuple<Tensor &,Tensor &> mode_out(Tensor& values, Tensor& indices,
            "mode only supports CPU AND CUDA backend, got: ", toString(self.type().backend()));
   dim = maybe_wrap_dim(dim, self.dim());
   if (_dimreduce_return_trivial_no_ident(values, self, dim, keepdim, "mode")) {
-    AT_ASSERT(values.dim() == 0);
-    indices.resize_({}).fill_(0);
+    if (self.dim() == 0){
```

Comment:
I'm not sure why this is changed -- wasn't this working before for mode?

Reply:
This change was made to handle the new case that arises due to the change in `aten/src/ATen/native/ReduceOpsUtils.h`.